src/applications/personalization/dashboard-2/components/health-care/HealthCare.jsx

```diff
@@ -83,16 +92,33 @@ const HealthCare = ({
       : 'Send a secure message to your health care team';
 
   return (
-    <div className="health-care vads-u-margin-y--6">
-      <h2 className="vads-u-margin-top--0 vads-u-margin-bottom--2">
+    <div className="health-care-wrapper vads-u-margin-y--6">
+      <h2 className="vads-u-margin-top--0 vads-u-margin-bottom--4">
         Health care
       </h2>
 
-      <div className="vads-u-display--flex medium-screen:vads-u-flex-direction--row vads-u-flex-direction--column">
-        {/* Appointments */}
-        <Appointments appointments={appointments} />
+      <div className="vads-u-display--flex large-screen:vads-u-flex-direction--row vads-u-flex-direction--column health-care">
+        {hasUpcomingAppointment && (
+          /* Appointments */
+          <Appointments appointments={appointments} />
+        )}
+
+        <div className="vads-u-display--flex vads-u-flex-direction--column large-screen:vads-u-flex--1">
+          {!hasUpcomingAppointment && (
+            <>
```

Comment:
If the user DOES have an appointment after 30 days, we display `You have no appointments scheduled in the next 30 days.` as well as the `Schedule and view your appointments` link. If the user does not have any scheduled appointments, all we show is the `Schedule and view your appointments` link.